architecture/additional_concepts/storage.adoc

```diff
@@ -20,11 +20,11 @@ specific knowledge of the underlying storage infrastructure.
 
 PVCs are specific to a
 link:../../architecture/core_concepts/projects_and_users.html#project[project]
-and are created and used by developers. A PVC is required if a developer wants
-to use a PV. The PVs themselves are not bound to a single project; they can be
-shared across the entire OpenShift cluster and claimed from any project. After a
-PV has been bound to a PVC, however, that PV cannot be bound to additional PVCs.
-This has the effect of scoping a bound PV to a single
+and are created and used by developers as a means to use a PV. PV resources on
+their own are not scoped to any single project; they can be shared across the
+entire OpenShift cluster and claimed from any project. After a PV has been
+link:#binding[_bound_] to a PVC, however, that PV cannot then be bound to
+additional PVCs. This has the effect of scoping a bound PV to a single
 link:../../architecture/core_concepts/projects_and_users.html#namespaces[namespace]
```

Comment:
Line 155: ifdef::openshift-enterprise,openshift-origin[]
In my view of this doc I see the following:
---------------------------------------------
   Types of Persistent Volumes
   OpenShift supports the following PersistentVolume plug-ins:

   Capacity
---------------------------------------------
In other words, the volume plug-ins are omitted. I think the ifdef should appear before "Types of Persistent Volumes"